src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -14,6 +14,43 @@ public static class ThreadPoolTests
         private const int ExpectedTimeoutMilliseconds = ThreadTestHelpers.ExpectedTimeoutMilliseconds;
         private const int MaxPossibleThreadCount = 0x7fff;
 
+        static ThreadPoolTests()
+        {
+            // Run the following tests before any others
+            ConcurrentInitializeTest();
+        }
+
+        [Fact]
```

Comment:
It's not necessary, but the invocation from the static constructor is only a workaround. It looks like xunit has test priorities to have tests run in a particular order but our version does not seem to support it. I would prefer to use that when it's available, for now the extra test invocation shouldn't hurt.